source/MaterialXTest/ShaderGen.cpp

```diff
@@ -1559,7 +1688,7 @@ TEST_CASE("Unique Names", "[shadergen]")
     // Generate a shade with an internal node having the same name as the shader,
     // which will result in a name conflict between the shader output and the
     // internal node output
-    const std::string shaderName = "foo";
+    const std::string shaderName = "unique_names";
```

Comment:
Change name so that oslc will generate a consistent named oso file with other outputted files (osl, frag, vert, ogsfx, mtlx).